src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/LegacyPackageReferenceProject.cs

```diff
@@ -156,6 +158,65 @@ private CentralPackageVersion ToCentralPackageVersion(string packageId, string v
             return new CentralPackageVersion(packageId, VersionRange.Parse(version));
         }
 
+        private RestoreAuditProperties GetRestoreAuditProperties()
+        {
+            ThreadHelper.ThrowIfNotOnUIThread();
+
+            string enableAudit = _vsProjectAdapter.BuildProperties.GetPropertyValue(ProjectBuildProperties.NuGetAudit);
+            string auditLevel = _vsProjectAdapter.BuildProperties.GetPropertyValue(ProjectBuildProperties.NuGetAuditLevel);
+            string auditMode = _vsProjectAdapter.BuildProperties.GetPropertyValue(ProjectBuildProperties.NuGetAuditMode);
+            HashSet<string> suppressedAdvisories = GetSuppressedAdvisories();
+
+            bool hasAnyValue = !string.IsNullOrWhiteSpace(enableAudit)
+                || !string.IsNullOrWhiteSpace(auditLevel)
+                || !string.IsNullOrWhiteSpace(auditMode)
+                || suppressedAdvisories?.Count > 0;
+
+            RestoreAuditProperties auditProperties = hasAnyValue
```

Comment:
Does this affect no-op, or is this skipped if there's no values?
What does CLI do?

Reply:
Sounds like this makes no-op consistent.

Based on the fact that we always set these properties however, this check is likely never going to be true.